src/app/page.tsx

```diff
@@ -7,65 +8,61 @@ import { ContentContainer } from './components/ContentContainer/ContentContainer
 import { homePageHero, homeContent } from './data/home';
+import { ImageCard } from './components/ImageCard/ImageCard';
 
-export default function Home() {
-  useHeroInit(homePageHero);
-  return (
-    <>
-      <DibbsSection />
-      <ValueSection />
-      <JurisdictionSection />
-      <InvitationCta />
-    </>
-  );
-}
+const commonTextStyles = {
+  heading: 'text-[1.5rem] font-bold text-[#224a58] xl:text-[2rem]',
+  paragraph: 'text-base font-normal leading-relaxed text-[#224a58]',
+};
 
-function DibbsSection() {
+const DibbsSection = memo(function DibbsSection() {
   return (
     <ContentContainer align>
-      <div className="grid grid-cols-1 justify-items-center gap-4 xl:grid-cols-[1fr_2fr] xl:justify-items-start xl:gap-14">
-        <div className="order-1">
-          <Image
-            className="xl:max-h-[20rem] xl:max-w-[30rem]"
-            src={`${basePath}/images/placeholder.png`}
-            width={480}
-            height={320}
-            alt="Placeholder"
-          />
-        </div>
-        <div className="order-2 flex flex-col gap-5">
-          <h2 className="min-w-full text-center text-[1.5rem] font-bold text-[#224a58] xl:text-left xl:text-[2rem]">
-            Introducing Data Integration Building Blocks
-          </h2>
-          <div className="flex flex-col gap-2">
-            <p className="m-0 max-w-[39.7rem] p-0 text-base font-normal leading-relaxed text-[#224a58]">
-              {homeContent.dibbs.description}
-            </p>
-            <ul className="text-base font-semibold leading-relaxed text-[#224a58]">
-              {homeContent.dibbs.benefits.map((benefit, index) => (
-                <li className="min-w-full" key={`benefit-${index}`}>
-                  {benefit}
-                </li>
-              ))}
-            </ul>
-          </div>
+      <ImageCard
+        imageUrl={`${basePath}/images/homepage-1.jpeg`}
+        imageAlt="Data Integration Building Blocks illustration"
+        imageFirst={true}
+        imageStyle={{
+          transform: 'scale(1.4) translate(-8%, 3%)',
+        }}
+      >
+        <h2
+          className={`min-w-full text-center ${commonTextStyles.heading} xl:text-left`}
+        >
+          Introducing Data Integration Building Blocks
+        </h2>
+        <div className="flex flex-col gap-2">
+          <p
+            className={`m-0 max-w-[39.7rem] p-0 ${commonTextStyles.paragraph}`}
+          >
+            {homeContent.dibbs.description}
+          </p>
+          <ul className={`${commonTextStyles.paragraph} font-semibold`}>
+            {homeContent.dibbs.benefits.map((benefit, index) => (
+              <li className="min-w-full" key={`benefit-${index}`}>
+                {benefit}
+              </li>
+            ))}
+          </ul>
         </div>
-      </div>
+      </ImageCard>
     </ContentContainer>
   );
-}
+});
 
-function ValueSection() {
+const ValueSection = memo(function ValueSection() {
```

Comment:
Any particular reason to switch these to using `memo`? React guidance generally suggests not to use this unless experiencing performance issues, which I imagine we should not be. Did you run into an issue?

Here's the snippet from the React docs I'm referring to (under `Deep Dive` [on this page](https://react.dev/reference/react/memo)):
<img width="873" alt="image" src="https://github.com/user-attachments/assets/dc324d4b-10fd-4e2b-9c25-07c935118147">

Reply:
No, it was just a suggestion by cursor